scripts/deployment_liquidity_controls.py

```diff
@@ -0,0 +1,196 @@
+import json
+
+from brownie import (
+    accounts,
+    ERC721,
+    ERC20,
+    LendingPoolCore,
+    LendingPoolPeripheral,
+    CollateralVaultCore,
+    CollateralVaultPeripheral,
+    LoansCore,
+    Loans,
+    LiquidationsCore,
+    LiquidationsPeripheral,
+    LiquidityControls,
+    web3
+)
+from pathlib import Path
+
+
+NAME_TO_CONTRACT = {
+    "collateral_vault_core": CollateralVaultCore,
+    "collateral_vault_peripheral": CollateralVaultPeripheral,
+    "lending_pool_core": LendingPoolCore,
+    "lending_pool": LendingPoolPeripheral,
+    "liquidations_peripheral": LiquidationsPeripheral,
+    "liquidations_core": LiquidationsCore,
+    "liquidity_controls": LiquidityControls,
+    "loans": Loans,
+    "loans_core": LoansCore,
+    "token": ERC20
+}
+
+
+def load_contracts(env: str) -> dict:
+    config_file = f"{Path.cwd()}/configs/{env}/contracts.json"
+    with open(config_file, "r") as f:
+        contracts = json.load(f)["tokens"]["WETH"]
+
+    return {key: NAME_TO_CONTRACT[key].at(contracts[key]["contract"]) for key in contracts}
+
+
+def load_nft_contracts(env: str) -> dict:
+    config_file = f"{Path.cwd()}/configs/{env}/nfts.json"
+    with open(config_file, "r") as f:
+        contracts = json.load(f)
+
+    return {
+        "cool_cats_instance": ERC721.at(contracts[0]["contract"]),
+        "hashmasks_instance": ERC721.at(contracts[1]["contract"]),
+        "kennel_instance": ERC721.at(contracts[2]["contract"]),
+        "doodles_instance": ERC721.at(contracts[3]["contract"]),
+        "wow_instance": ERC721.at(contracts[4]["contract"]),
+        "mutant_instance": ERC721.at(contracts[5]["contract"]),
+        "veefriends_instance": ERC721.at(contracts[6]["contract"]),
+        "pudgypenguins_instance": ERC721.at(contracts[7]["contract"]),
+        "bayc_instance": ERC721.at(contracts[8]["contract"]),
+        "wpunks_instance": ERC721.at(contracts[9]["contract"])
+    }
+
+
+def dev():
```

Comment:
See my previous comment about dev(), int() and prod() being replaced by one function.